source/beamline/AMPVControl.cpp

```diff
@@ -213,7 +213,7 @@ AMControl::FailureExplanation AMPVControl::move(double setpoint) {
 void AMPVControl::onNewFeedbackValue(double) {
 
 	// If we're not in the middle of a move, don't really care about changing values.
-	if(!moveInProgress_)
+	if(!moveInProgress_ && !completionTimer_.isActive())
```

Comment:
There is another issue if your crystal selection move is taking too long.  We need to think about how these changes can affect other implementations and a general change to AMPVControl for a single exception does not seem like a good idea to me.